src/api/search/api-search-schemas.ts

```diff
@@ -66,6 +66,12 @@ export type DocumentLinkBody = Static<typeof documentLinkBodySchema>;
 
 export const searchBodySchema = Type.Object(
   {
+    enhanceSimilarity: Type.Optional(
+      Type.Boolean({
+        default: false,
+        description: 'Enhance the semantic search',
+      }),
+    ),
```

Comment:
I'd change this to `prefer` (or a better word if you find one);

like this:
```typescript
prefer: 'speed' | 'accuracy'
```

where, behind the scenes, we translate "speed" by not using Hyde, an accuracy using it


ps: can you measure the amount of added time to use Hyde? What's the impact on performance? I'd guess it's around 500ms - 1s ? This is important to document it as well (there's a description field in Typebox)

Reply:
and add a default to speed